release/benchmarks/object_store.yaml

```diff
@@ -5,14 +5,14 @@ max_workers: 49
 
 head_node_type:
     name: head_node
-    instance_type: m4.16xlarge
+    instance_type: m5.16xlarge
     resources:
       custom_resources:
         node: 1
 
 worker_node_types:
     - name: worker_node
-      instance_type: m4.2xlarge
```

Comment:
Let's see how the perf number changes after this

Reply:
not blocking, but why not jump to m6 directly?

Reply:
There does seem to be a perf improvement
perf_metrics = [{'perf_metric_name': 'time_to_broadcast_1073741824_bytes_to_50_nodes', 'perf_metric_value': 17.215506515, 'perf_metric_type': 'LATENCY'}]

It's 17.2 and the 30-day best before was 22
 

<br class="Apple-interchange-newline">

Reply:
> not blocking, but why not jump to m6 directly?

open to going to m6, looks like m5 is still pretty old 2017, while m6 is 2021. m6i looks like the equivalent but the perf metrics might change more

Reply:
ack, upgrading to m6i's, m7i's possibly too new, no other tests we have are using them and are slightly more expensive